### ISSUE TITLE
GitHub Enterprise support

### ISSUE THREAD

**Issue by rwkarg** [timheuer/GitHubActionsVS#27]
Allow configuration of GitHub Enterprise address that would be additionally searched for in [RepoInfo.cs](https://github.com/timheuer/GitHubActionsVS/blob/a68954fe72bd8c163c87a7fc9afbdf65a98b227c/src/Helpers/RepoInfo.cs#L26)
https://github.com/timheuer/GitHubActionsVS/blob/a68954fe72bd8c163c87a7fc9afbdf65a98b227c/src/Helpers/RepoInfo.cs#L26

**Comment by timheuer** [comment]
Thanks @rwkarg for the note. I do not have a GHE account.  Can you help educate me a bit here?
- Would the URL be fully custom? (like it could be `some.cool.url.xyz`)?
- Is the authentication scheme different? Meaning you authenticate against that URI scheme as well

After these answers would you be willing to test out an example of this if I put it in a diff branch?

**Comment by rwkarg** [comment]
Url for a repository would be something like `https://internal-repository.my-corp.net/org/repo` where the `internal-repository.my-corp.net` seems to be the bit that would get configured (and maybe a port too? I don't need to configure that but others might?).

The Oktokit SDK has worked for everything I've used it for against Enterprise so it should be fine 🤞.

If I can figure out how to get it installed, then I'd be happy to give a branch a try!

**Comment by timheuer** [comment]
Ok, thanks @rwkarg. I'll see if I can spend a bit on this looking into it.

**Comment by msun03** [comment]
So GitHub Enterprise is not supported currently?

**Comment by timheuer** [comment]
> So GitHub Enterprise is not supported currently?

Correct. I haven’t had time to look into this and even get a GHE setup.
